examples/evm/coverage.py

```diff
@@ -11,14 +11,14 @@ def transaction(self, transaction):
         return transaction
 
     def __init__(self):
-        super(ManticoreEVM, self).__init__()
         #Make the constraint store
         constraints = ConstraintSet()
         #make the ethereum world state
         world = evm.EVMWorld(constraints)
 
         #Let's start with the symbols
         self.temp_initial_state = State(constraints, world)
+        super(ManticoreEVM, self).__init__(self.temp_initial_state)
```

Comment:
Move `super()` to last line of the ctor? ctor can either establish state for subclass, or depend on it, keeping it in the middle can potentially be confusing.